libraries/AP_Airspeed/AP_Airspeed_config.h

```diff
@@ -71,3 +71,7 @@
 #ifndef AP_AIRSPEED_EXTERNAL_ENABLED
 #define AP_AIRSPEED_EXTERNAL_ENABLED AP_AIRSPEED_ENABLED && HAL_EXTERNAL_AHRS_ENABLED
 #endif
+
+#ifndef AP_AIRSPEED_ND210_ENABLED
+#define AP_AIRSPEED_ND210_ENABLED AP_AIRSPEED_ENABLED
```

Comment:
should this be changed to `AP_AIRSPEED_BACKEND_DEFAULT_ENABLED` instead of `AP_AIRSPEED_ENABLED` like it is in line 26 with the DLVR sensor? I don't really understand why there is this distinction, as they are both set to the same value in line 13.